### PR TITLE
content: update annuaire entreprise

### PR DESCRIPTION
**Pull request by XavierJp**
- update Karen and Yoan
- update page

### CODE REVIEW

File: content/_startups/annuaire-entreprises.md [hunk 1]
```diff
@@ -25,12 +27,14 @@ thematiques:
 ---
 ## Le problème :
 
-Au sein de l’administration, la donnée des entreprises françaises est une ressource dispersée. De nombreuses directions, agences ou instituts sont responsables d'une fraction de la données et en assument la publication.
+Au sein de l’administration, la donnée des entreprises est une ressource dispersée. De nombreuses directions, agences ou instituts sont responsables d'une fraction de la données et en assument la publication.
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:30]:
```suggestion
Au sein de l’administration, la donnée des entreprises est une ressource dispersée. De nombreuses directions, agences ou instituts sont responsables d'une fraction de cette donnée et en assurent la publication.
```

File: content/_startups/annuaire-entreprises.md [hunk 2]
```diff
@@ -25,12 +27,14 @@ thematiques:
...
 
-Résultat, les citoyens, entrepreneurs et agents qui ont besoin d'y accèder doivent choisir entre :
+Résultat, les citoyens, entrepreneurs et agents publics doivent choisir entre :
 
--   les acteurs privés qui centralisent les données, font payer une partie des données, et qui ne sont pas garants de son exactitude
--   les acteurs publics officiels, chacun garant de certaines données, avec chacun leur site internet, entre lesquels il faut savoir naviguer (INSEE, INPI, DILA, Ministère de l’Intérieur)
+-   les acteurs privés qui centralisent les données et qui en font payer une partie sans pouvoir en garantir l’exactitude
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:34]:
```suggestion
-   les acteurs privés qui centralisent les données et en facturent une partie sans pouvoir en garantir l’exactitude
```

File: content/_startups/annuaire-entreprises.md [hunk 3]
```diff
@@ -25,12 +27,14 @@ thematiques:
...
+-   les acteurs publics officiels, chacun garant d’une ou plusieurs bases de données, avec chacun ses modalités d’accès (sites, jeux de données, API)
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:35]:
```suggestion
-   les acteurs publics officiels, chacun garant d’une ou plusieurs bases de données, avec ses propres modalités d’accès (sites, jeux de données, API)
```

File: content/_startups/annuaire-entreprises.md [hunk 4]
```diff
@@ -25,12 +27,14 @@ thematiques:
...
+
+Cette complexité est une perte de temps et d’efficacité pour les relations entre les entreprises et l’administrations.
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:37]:
```suggestion
Cette complexité est une perte de temps et d’efficacité pour les relations entre les entreprises et l’administration.
```

File: content/_startups/annuaire-entreprises.md [hunk 5]
```diff
@@ -39,4 +43,8 @@ Résultat, les citoyens, entrepreneurs et agents qui ont besoin d'y accèder doi
 -   un moteur de recherche d'entreprise pour retrouver facilement n’importe quelle entreprise, association ou administration dotée d'un numéro siren
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:43]:
```suggestion
-   un moteur de recherche d'entreprises pour retrouver facilement n’importe quelle entreprise, association ou administration dotée d'un numéro siren
```

File: content/_startups/annuaire-entreprises.md [hunk 6]
```diff
@@ -39,4 +43,8 @@ Résultat, les citoyens, entrepreneurs et agents qui ont besoin d'y accèder doi
...
 -   pour chaque entreprise, une page identité avec les données récupérées en temps réel auprès des différentes sources de données "officielles"
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:44]:
```suggestion
-   pour chaque entreprise, une page d'identité avec les données récupérées en temps réel auprès des différentes sources de données officielles
```

File: content/_startups/annuaire-entreprises.md [hunk 7]
```diff
@@ -39,4 +43,8 @@ Résultat, les citoyens, entrepreneurs et agents qui ont besoin d'y accèder doi
...
 
-Mais l’Annuaire souhaite aller encore plus loin et devenir le site de référence de la donnée des entreprises : exhaustif, officiel, fiable et ergonomique
\ No newline at end of file
+L’Annuaire des Entreprises propose un [accès dédié pour les agents publics](https://annuaire-entreprises.data.gouv.fr/lp/agent-public). En s’identifiant, les agents bénéficient d’un accès complet aux informations des entreprises, pas uniquement les informations publiques.
```
Comment (HAEKADI) [inline content/_startups/annuaire-entreprises.md:46]:
```suggestion
L’Annuaire des Entreprises propose un [accès dédié pour les agents publics](https://annuaire-entreprises.data.gouv.fr/lp/agent-public). En s’identifiant, les agents bénéficient d’un accès complet aux informations des entreprises, et pas uniquement aux informations publiques. 
```

**Review by HAEKADI — APPROVED** [review]
lgtm 🚀

**Comment by XavierJp** [comment]
Merci Hajar !

**Pull request closed without merging**